src/pages/settings/Profile/CustomStatus/SetTimePage.js

```diff
@@ -2,7 +2,7 @@ import lodashGet from 'lodash/get';
 import React from 'react';
 import {View} from 'react-native';
 import {withOnyx} from 'react-native-onyx';
-import FormProvider from '@components/Form/FormProvider';
+import Form from '@components/Form';
```

Comment:
@perunt Was it not possible to get this working with `FormProvider`? I _think_ we're meant to use it everywhere with functional components now. Confirming on [Slack](https://expensify.slack.com/archives/C01GTK53T8Q/p1699535496890069).

Reply:
[Confirmed](https://expensify.slack.com/archives/C01GTK53T8Q/p1699537646546139?thread_ts=1699535496.890069&cid=C01GTK53T8Q) that we should be using it for all new components.

Reply:
Didn't know that. Thanks for pointing it up, as this pr was made before we started to use this `FormProvider`.

Btw, I no longer use the form there, so it's an outdated link. Can you pull the new date?